api/python/src/cellxgene_ontology_guide/ontology_parser.py

```diff
@@ -664,3 +664,60 @@ def get_term_id_by_label(self, term_label: str, ontology_name: str) -> Optional[
         """
         ontology_term_label_to_id_map = self.get_term_label_to_id_map(ontology_name)
         return ontology_term_label_to_id_map.get(term_label)
+
+    def get_bridge_term_id(self, term_id: str, cross_ontology: str) -> Optional[str]:
+        """
+        For a given term ID, fetch the equivalent term ID from a given ontology. Only returns exact match if it exists.
+
+        If no applicable match is found, returns None.
+
+        Raises ValueError if term ID or cross_ontology are not valid member of a supported ontology.
+
+        Example
+        >>> from cellxgene_ontology_guide.ontology_parser import OntologyParser
+        >>> ontology_parser = OntologyParser()
+        >>> ontology_parser.get_bridge_term_id("FBbt:00000001", "UBERON")
+        'UBERON:0000468'
+
+        :param term_id: str ontology term to find equivalent term for
+        :param cross_ontology: str name of ontology to search for equivalent term in
+        :return: Optional[str] equivalent term ID from the cross_ontology
+        """
+        # TODO: best way to communicate that we only support querying bridge terms for certain ontologies
+        # (i.e. ZFS/FBBT -> CL/UBERON) and not all of our supported ontologies?
+        ontology_name = self._parse_ontology_name(term_id)
+        bridge_term_id: Optional[str] = self.cxg_schema.ontology(ontology_name)[term_id]["cross_ontology_terms"].get(
```

Comment:
seems reasonable to just wrap this in a try/except and raise a more descriptive error message